backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/processing/ImportProcessTemplate.java

```diff
@@ -716,4 +731,12 @@ private DataImportConfigId extractDataImportConfigIdOrNull(@NonNull final Import
 		final Optional<Integer> value = InterfaceWrapperHelper.getValue(importRecord, importTableDescriptor.getDataImportConfigIdColumnName());
 		return value.map(DataImportConfigId::ofRepoIdOrNull).orElse(null);
 	}
+
+	private DataImportRunId extractDataImporRunIdOrNull()
```

Comment:
pls avoid this...

why not better
option 1. (preferred) propagate DataImportRunId from DataImportCommand
option 2. (workaround-ish and might not cover all cases) provide AD_PInstance_ID to those DB functions.